diff -Nru libindicate-0.6.92.orig/Makefile.am libindicate-0.6.92/Makefile.am
--- libindicate-0.6.92.orig/Makefile.am	2012-06-05 13:23:17.037530346 -0400
+++ libindicate-0.6.92/Makefile.am	2012-06-05 13:23:32.100519149 -0400
@@ -26,7 +26,7 @@
 	xmldocs.make \
 	autogen.sh
 
-DISTCHECK_CONFIGURE_FLAGS = --enable-gtk-doc --disable-scrollkeeper --disable-introspection
+DISTCHECK_CONFIGURE_FLAGS = --enable-gtk-doc --disable-scrollkeeper
 
 DISTCLEANFILES = \
 	libindicate-*.tar.gz \
