diff -Nru libqtgconf-0.1.orig/CMakeLists.txt libqtgconf-0.1/CMakeLists.txt
--- libqtgconf-0.1.orig/CMakeLists.txt	2012-07-03 23:26:00.847405138 -0400
+++ libqtgconf-0.1/CMakeLists.txt	2012-07-03 23:26:29.898433929 -0400
@@ -80,7 +80,7 @@
     )
 
 # Install
-set(IMPORT_INSTALL_DIR lib/qt4/imports/gconf)
+set(IMPORT_INSTALL_DIR lib${LIB_SUFFIX}/qt4/imports/gconf)
 set(INCLUDE_INSTALL_DIR include/QtGConf)
 
 ## QtGConf
@@ -94,7 +94,7 @@
 
 # FIXME: Generate from ${CMAKE_INSTALL_PREFIX}
 install(FILES libqtgconf.pc
-    DESTINATION lib/pkgconfig
+    DESTINATION lib${LIB_SUFFIX}/pkgconfig
     )
 
 ## QtGConfQml
