--- a/src/signon-oauth2plugin.pc
+++ b/src/signon-oauth2plugin.pc
@@ -1,9 +1,9 @@
 prefix=/usr
 exec_prefix=${prefix}
-libdir=${prefix}/lib/
+libdir=${prefix}/@LIB@/
 includedir=${prefix}/include
 Name: signon-oauth2plugin
 Description: Signon OAuth 2.0 plugin
 Version: 0.0.1
 Requires: signon-plugins
-Libs.private: -L/usr/lib  -lQtCore
+Libs.private: -L/usr/@LIB@  -lQtCore
--- a/src/src.pro
+++ b/src/src.pro
@@ -2,7 +2,7 @@
 include( ../common-vars.pri )
 TEMPLATE = lib
 TARGET = oauth2plugin
-DESTDIR = lib/signon
+DESTDIR = @LIB@/signon
 QT += core \
     network \
     xmlpatterns
