diff -Nru gnome-session-3.4.2.orig/gnome-session/gsm-manager.c gnome-session-3.4.2/gnome-session/gsm-manager.c
--- gnome-session-3.4.2.orig/gnome-session/gsm-manager.c	2012-07-16 14:19:33.817145645 -0400
+++ gnome-session-3.4.2/gnome-session/gsm-manager.c	2012-07-16 14:24:03.705084389 -0400
@@ -1607,7 +1607,6 @@
                 do_phase_startup (manager);
                 break;
         case GSM_MANAGER_PHASE_RUNNING:
-                possibly_show_fallback_dialog (manager);
                 g_signal_emit (manager, signals[SESSION_RUNNING], 0);
                 update_idle (manager);
                 break;
