diff -Nru indicator-datetime-0.3.94.orig/src/Makefile.am indicator-datetime-0.3.94/src/Makefile.am
--- indicator-datetime-0.3.94.orig/src/Makefile.am	2012-07-17 00:32:06.783226850 -0400
+++ indicator-datetime-0.3.94/src/Makefile.am	2012-07-17 00:42:13.278419141 -0400
@@ -20,7 +20,7 @@
 	-Werror \
 	$(SERVICE_CFLAGS) \
 	$(COVERAGE_CFLAGS) \
-	-DTIMEZONE_FILE="\"/etc/timezone\"" \
+	-DTIMEZONE_FILE="\"/etc/sysconfig/clock\"" \
 	-DG_LOG_DOMAIN=\"Indicator-Datetime\"
 indicator_datetime_service_LDADD = \
 	$(SERVICE_LIBS)
@@ -40,7 +40,7 @@
 	$(INDICATOR_CFLAGS) \
 	$(COVERAGE_CFLAGS) \
 	-Wall -Werror \
-	-DTIMEZONE_FILE="\"/etc/timezone\"" \
+	-DTIMEZONE_FILE="\"/etc/sysconfig/clock\"" \
 	-DG_LOG_DOMAIN=\"Indicator-Datetime\"
 libdatetime_la_LIBADD = \
 	$(INDICATOR_LIBS)
@@ -62,7 +62,7 @@
 	-Werror \
 	$(PREF_CFLAGS) \
 	$(COVERAGE_CFLAGS) \
-	-DTIMEZONE_FILE="\"/etc/timezone\"" \
+	-DTIMEZONE_FILE="\"/etc/sysconfig/clock\"" \
 	-DPKGDATADIR="\"$(pkgdatadir)\""
 libindicator_datetime_la_LIBADD = \
 	$(PREF_LIBS)
diff -Nru indicator-datetime-0.3.94.orig/src/utils.c indicator-datetime-0.3.94/src/utils.c
--- indicator-datetime-0.3.94.orig/src/utils.c	2012-07-17 00:32:06.782226847 -0400
+++ indicator-datetime-0.3.94/src/utils.c	2012-07-17 00:34:13.532265461 -0400
@@ -121,24 +121,65 @@
 read_timezone ()
 {
 	GError * error = NULL;
-	gchar * tempzone = NULL;
-	if (!g_file_get_contents(TIMEZONE_FILE, &tempzone, NULL, &error)) {
+	gchar * actual_timezone = NULL;
+	gchar * file_contents = NULL;
+	if (!g_file_get_contents(TIMEZONE_FILE, &file_contents, NULL, &error)) {
 		g_warning("Unable to read timezone file '" TIMEZONE_FILE "': %s", error->message);
 		g_error_free(error);
 		return NULL;
 	}
 
+	/* Read file line by line */
+	gchar ** file_contents_split = g_strsplit(file_contents, "\n", 0);
+
+	int i;
+	for (i = 0; i < g_strv_length(file_contents_split); i++) {
+		if (file_contents_split[i][0] != '#') {
+			/* Find line where the timezone is defined */
+			if (g_strstr_len(file_contents_split[i], -1, "ZONE=") != NULL) {
+				/* Value is on the right side of the equals sign when setting bash variables */
+				gchar ** timezone_line_split = g_strsplit(file_contents_split[i], "=", 0);
+
+				/* In bash syntax, if quotation marks are used, it must be right next to the equals sign */
+				if(timezone_line_split[1][0] == '\'' ||
+				   timezone_line_split[1][0] == '"') {
+					gchar ** timezone_quoted_split = g_strsplit(timezone_line_split[1], "\"", 0);
+					actual_timezone = g_strdup(timezone_quoted_split[1]);
+					g_strfreev(timezone_quoted_split);
+
+					/* No need to continue */
+					g_strfreev(timezone_line_split);
+					break;
+				}
+				else {
+					actual_timezone = g_strdup(timezone_line_split[1]);
+
+					/* No need to continue */
+					g_strfreev(timezone_line_split);
+					break;
+				}
+
+				g_strfreev(timezone_line_split);
+			}
+		}
+	}
+
+	g_strfreev(file_contents_split);
+
 	/* This shouldn't happen, so let's make it a big boom! */
-	g_return_val_if_fail(tempzone != NULL, NULL);
+	g_return_val_if_fail(actual_timezone != NULL, NULL);
 
-	/* Note: this really makes sense as strstrip works in place
-	   so we end up with something a little odd without the dup
-	   so we have the dup to make sure everything is as expected
-	   for everyone else. */
-	gchar * rv = g_strdup(g_strstrip(tempzone));
-	g_free(tempzone);
+	g_free(file_contents);
+
+	/* No idea why Fedora uses spaces when tzdata uses underscores */
+	for (i = 0; i < strlen(actual_timezone); i++) {
+		if (actual_timezone[i] == ' ') {
+			actual_timezone[i] = '_';
+		}
+	}
+	//g_strcanon(actual_timezone, "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ/", '_');
 
-  return rv;
+	return actual_timezone;
 }
 
 /* Translate msg according to the locale specified by LC_TIME */
