diff -Nru unity-lens-videos-0.3.10.orig/setup.py unity-lens-videos-0.3.10/setup.py
--- unity-lens-videos-0.3.10.orig/setup.py	2012-10-01 17:27:46.856775864 -0400
+++ unity-lens-videos-0.3.10/setup.py	2012-10-01 17:28:15.401766908 -0400
@@ -10,7 +10,7 @@
       url="http://launchpad.net/~davidc3",
       license="GNU General Public License (GPL)",
       data_files=[
-    ('lib/unity-lens-video', ['src/unity-lens-video']),
+    ('libexec', ['src/unity-lens-video']),
     ('share/dbus-1/services', ['unity-lens-video.service']),
     ('share/applications', ['unity-lens-video.desktop']),
     ('share/pixmaps', ['unity-lens-video.png']),
diff -Nru unity-lens-videos-0.3.10.orig/unity-lens-video.service unity-lens-videos-0.3.10/unity-lens-video.service
--- unity-lens-videos-0.3.10.orig/unity-lens-video.service	2012-10-01 17:27:46.853775862 -0400
+++ unity-lens-videos-0.3.10/unity-lens-video.service	2012-10-01 17:28:27.931762539 -0400
@@ -1,3 +1,3 @@
 [D-BUS Service]
 Name=net.launchpad.lens.video
-Exec=/usr/lib/unity-lens-video/unity-lens-video
+Exec=/usr/libexec/unity-lens-video
