diff -up gtk+-2.24.14/configure.in.automak gtk+-2.24.14/configure.in
--- gtk+-2.24.14/configure.in.automak	2013-01-15 18:29:09.834915289 -0500
+++ gtk+-2.24.14/configure.in	2013-01-15 18:30:19.857912786 -0500
@@ -44,14 +44,13 @@ AC_INIT([gtk+], [gtk_version],
 	[gtk+])
 
 AC_CONFIG_SRCDIR([gdk/gdktypes.h])
-
+AC_CONFIG_HEADERS(config.h)
 AC_CONFIG_MACRO_DIR([m4])
 
 # Save this value here, since automake will set cflags later
 cflags_set=${CFLAGS+set}
 
 AM_INIT_AUTOMAKE([no-define -Wno-portability dist-bzip2])
-AM_CONFIG_HEADER(config.h)
 
 # Support silent build rules, requires at least automake-1.11. Enable
 # by either passing --enable-silent-rules to configure or passing V=0
@@ -300,7 +299,6 @@ AM_SANITY_CHECK
 
 # Checks for programs.
 AC_ISC_POSIX
-AM_PROG_CC_STDC
 AM_PROG_CC_C_O
 AC_PROG_INSTALL
 AC_PROG_MAKE_SET
