diff -Nru network-manager-applet-0.9.7.0.orig/src/applet.c network-manager-applet-0.9.7.0/src/applet.c
--- network-manager-applet-0.9.7.0.orig/src/applet.c	2012-09-23 18:31:23.785962212 -0400
+++ network-manager-applet-0.9.7.0/src/applet.c	2012-09-23 18:32:27.333975562 -0400
@@ -1857,15 +1857,15 @@
 
 	n_wifi = nma_menu_add_devices (menu, applet);
 
-	nma_menu_add_vpn_submenu (menu, applet);
-
 	if (n_wifi > 0 && nm_client_wireless_get_enabled (applet->nm_client)) {
 		/* Add the "Hidden Wi-Fi network..." entry */
-		nma_menu_add_separator_item (menu);
 		nma_menu_add_hidden_network_item (menu, applet);
 		nma_menu_add_create_network_item (menu, applet);
+		nma_menu_add_separator_item (menu);
 	}
 
+	nma_menu_add_vpn_submenu (menu, applet);
+
 	gtk_widget_show_all (menu);
 
 //	nmi_dbus_signal_user_interface_activated (applet->connection);
