diff -Nru network-manager-applet-0.9.7.0.orig/nm-applet.convert network-manager-applet-0.9.7.0/nm-applet.convert
--- network-manager-applet-0.9.7.0.orig/nm-applet.convert	2012-09-23 18:29:01.678920187 -0400
+++ network-manager-applet-0.9.7.0/nm-applet.convert	2012-09-23 18:37:53.380071028 -0400
@@ -5,3 +5,4 @@
 suppress-wireless-networks-available = /apps/nm-applet/suppress-wireless-networks-available
 stamp = /apps/nm-applet/stamp
 disable-wifi-create = /apps/nm-applet/disable-wifi-create
+show-applet = /apps/nm-applet/show-applet
diff -Nru network-manager-applet-0.9.7.0.orig/org.gnome.nm-applet.gschema.xml.in network-manager-applet-0.9.7.0/org.gnome.nm-applet.gschema.xml.in
--- network-manager-applet-0.9.7.0.orig/org.gnome.nm-applet.gschema.xml.in	2012-09-23 18:29:01.652920161 -0400
+++ network-manager-applet-0.9.7.0/org.gnome.nm-applet.gschema.xml.in	2012-09-23 18:38:46.329082528 -0400
@@ -31,6 +31,11 @@
       <_summary>Disable WiFi Create</_summary>
       <_description>Set to true to disable creation of adhoc networks when using the applet.</_description>
     </key>
+    <key name="show-applet" type="b">
+      <default>true</default>
+      <_summary>Show the applet in notification area</_summary>
+      <_description>Set to FALSE to disable displaying the applet in the notification area.</_description>
+    </key>
   </schema>
   <schema id="org.gnome.nm-applet.eap">
     <key name="ignore-ca-cert" type="b">
diff -Nru network-manager-applet-0.9.7.0.orig/src/applet.c network-manager-applet-0.9.7.0/src/applet.c
--- network-manager-applet-0.9.7.0.orig/src/applet.c	2012-09-23 18:32:43.927979787 -0400
+++ network-manager-applet-0.9.7.0/src/applet.c	2012-09-23 18:36:37.216043400 -0400
@@ -2649,6 +2649,8 @@
 	if (!nm_running)
 		state = NM_STATE_UNKNOWN;
 
+	gtk_status_icon_set_visible (applet->status_icon, applet->visible);
+
 	switch (state) {
 	case NM_STATE_UNKNOWN:
 	case NM_STATE_ASLEEP:
@@ -3349,6 +3351,21 @@
 	return success;
 }
 
+static void
+applet_gsettings_show_changed (GSettings *settings,
+                               gchar *key,
+                               gpointer user_data)
+{
+	NMApplet *applet = NM_APPLET (user_data);
+
+	g_return_if_fail (NM_IS_APPLET(applet));
+	g_return_if_fail (key != NULL);
+
+	applet->visible = g_settings_get_boolean (settings, key);
+
+	gtk_status_icon_set_visible (applet->status_icon, applet->visible);
+}
+
 static GObject *
 constructor (GType type,
              guint n_props,
@@ -3371,6 +3388,9 @@
 	}
 
 	applet->gsettings = g_settings_new (APPLET_PREFS_SCHEMA);
+	applet->visible = g_settings_get_boolean (applet->gsettings, PREF_SHOW_APPLET);
+	g_signal_connect (applet->gsettings, "changed::show-applet",
+	                  G_CALLBACK (applet_gsettings_show_changed), applet);
 
 	/* Load pixmaps and create applet widgets */
 	if (!setup_widgets (applet))
diff -Nru network-manager-applet-0.9.7.0.orig/src/applet.h network-manager-applet-0.9.7.0/src/applet.h
--- network-manager-applet-0.9.7.0.orig/src/applet.h	2012-09-23 18:29:01.775920280 -0400
+++ network-manager-applet-0.9.7.0/src/applet.h	2012-09-23 18:37:33.105063876 -0400
@@ -65,6 +65,7 @@
 #define PREF_DISABLE_VPN_NOTIFICATIONS            "disable-vpn-notifications"
 #define PREF_DISABLE_WIFI_CREATE                  "disable-wifi-create"
 #define PREF_SUPPRESS_WIFI_NETWORKS_AVAILABLE     "suppress-wireless-networks-available"
+#define PREF_SHOW_APPLET                          "show-applet"
 
 #define ICON_LAYER_LINK 0
 #define ICON_LAYER_VPN 1
@@ -95,6 +96,8 @@
 
 	GSettings *gsettings;
 
+	gboolean visible;
+
 	/* Permissions */
 	NMClientPermissionResult permissions[NM_CLIENT_PERMISSION_LAST + 1];
 
