diff -Nru unity-lens-photos-0.5.orig/setup.py unity-lens-photos-0.5/setup.py
--- unity-lens-photos-0.5.orig/setup.py	2012-09-14 15:34:19.270199391 -0400
+++ unity-lens-photos-0.5/setup.py	2012-09-14 15:34:48.649198582 -0400
@@ -10,11 +10,11 @@
       url="http://launchpad.net/unity-lens-photos",
       license="GNU General Public License (GPL)",
       data_files=[
-    ('lib/unity-lens-photos', ['src/unity-lens-photos']),
-    ('lib/unity-lens-photos', ['src/shotwell_scope.py']),
-    ('lib/unity-lens-photos', ['src/flickr_scope.py']),
-    ('lib/unity-lens-photos', ['src/facebook_scope.py']),
-    ('lib/unity-lens-photos', ['src/picasa_scope.py']),
+    ('libexec/unity-lens-photos', ['src/unity-lens-photos']),
+    ('libexec/unity-lens-photos', ['src/shotwell_scope.py']),
+    ('libexec/unity-lens-photos', ['src/flickr_scope.py']),
+    ('libexec/unity-lens-photos', ['src/facebook_scope.py']),
+    ('libexec/unity-lens-photos', ['src/picasa_scope.py']),
     ('share/dbus-1/services', ['unity-lens-photos.service']),
     ('share/accounts/applications', ['unity-lens-photos.application']),
     ('share/applications', ['unity-lens-photos.desktop']),
