diff -Nru unity-lens-photos-0.6.orig/setup.py unity-lens-photos-0.6/setup.py
--- unity-lens-photos-0.6.orig/setup.py	2012-09-20 19:35:33.884734770 -0400
+++ unity-lens-photos-0.6/setup.py	2012-09-20 19:35:58.742720858 -0400
@@ -11,11 +11,11 @@
       url="http://launchpad.net/unity-lens-photos",
       license="GNU General Public License (GPL)",
       data_files=[
-    ('lib/unity-lens-photos', ['src/unity-lens-photos']),
-    ('lib/unity-lens-photos', ['src/shotwell_scope.py']),
-    ('lib/unity-lens-photos', ['src/flickr_scope.py']),
-    ('lib/unity-lens-photos', ['src/facebook_scope.py']),
-    ('lib/unity-lens-photos', ['src/picasa_scope.py']),
+    ('libexec/unity-lens-photos', ['src/unity-lens-photos']),
+    ('libexec/unity-lens-photos', ['src/shotwell_scope.py']),
+    ('libexec/unity-lens-photos', ['src/flickr_scope.py']),
+    ('libexec/unity-lens-photos', ['src/facebook_scope.py']),
+    ('libexec/unity-lens-photos', ['src/picasa_scope.py']),
     ('share/unity-lens-photos/media', ['media/Flickr.svg']),
     ('share/unity-lens-photos/media', ['media/Facebook.svg']),
     ('share/unity-lens-photos/media', ['media/Picasa.svg']),
