diff -Nru unity-lens-photos-0.8.orig/setup.py unity-lens-photos-0.8/setup.py
--- unity-lens-photos-0.8.orig/setup.py	2012-10-03 23:49:17.261407961 -0400
+++ unity-lens-photos-0.8/setup.py	2012-10-03 23:50:02.473319576 -0400
@@ -11,11 +11,11 @@
       url="http://launchpad.net/unity-lens-photos",
       license="GNU General Public License (GPL)",
       data_files=[
-    ('lib/unity-lens-photos', ['src/unity-lens-photos']),
-    ('lib/unity-lens-photos', ['src/shotwell_scope.py']),
-    ('lib/unity-lens-photos', ['src/flickr_scope.py']),
-    ('lib/unity-lens-photos', ['src/facebook_scope.py']),
-    ('lib/unity-lens-photos', ['src/picasa_scope.py']),
+    ('libexec/unity-lens-photos', ['src/unity-lens-photos']),
+    ('libexec/unity-lens-photos', ['src/shotwell_scope.py']),
+    ('libexec/unity-lens-photos', ['src/flickr_scope.py']),
+    ('libexec/unity-lens-photos', ['src/facebook_scope.py']),
+    ('libexec/unity-lens-photos', ['src/picasa_scope.py']),
     ('share/dbus-1/services', ['unity-lens-photos.service']),
     ('share/applications', ['unity-lens-photos.desktop']),
     ('share/pixmaps', ['unity-lens-photos.png']),
diff -Nru unity-lens-photos-0.8.orig/unity-lens-photos.service unity-lens-photos-0.8/unity-lens-photos.service
--- unity-lens-photos-0.8.orig/unity-lens-photos.service	2012-10-03 23:49:17.251407976 -0400
+++ unity-lens-photos-0.8/unity-lens-photos.service	2012-10-03 23:50:44.286243844 -0400
@@ -1,3 +1,3 @@
 [D-BUS Service]
 Name=net.launchpad.lens.photos
-Exec=/usr/lib/unity-lens-photos/unity-lens-photos
+Exec=/usr/libexec/unity-lens-photos/unity-lens-photos
