diff -Nru signon-8.44.orig/lib/plugins/signon-plugins-common/signon-plugins-common.pc.in signon-8.44/lib/plugins/signon-plugins-common/signon-plugins-common.pc.in
--- signon-8.44.orig/lib/plugins/signon-plugins-common/signon-plugins-common.pc.in	2012-11-18 14:07:28.652921000 -0500
+++ signon-8.44/lib/plugins/signon-plugins-common/signon-plugins-common.pc.in	2012-11-18 14:07:57.166960000 -0500
@@ -6,5 +6,5 @@
 Name: signon-plugins-common
 Description: SignonPluginsCommon
 Version: PROJECT_VERSION
-Libs.private: -L/usr/lib  -lQtCore
+Libs.private: -L/usr/@LIB@  -lQtCore
 Cflags:  -I${includedir}/signon-plugins -I${includedir}/signon-plugins/SignOn
diff -Nru signon-8.44.orig/lib/plugins/signon-plugins.pc.in signon-8.44/lib/plugins/signon-plugins.pc.in
--- signon-8.44.orig/lib/plugins/signon-plugins.pc.in	2012-11-18 14:07:28.651639000 -0500
+++ signon-8.44/lib/plugins/signon-plugins.pc.in	2012-11-18 14:08:19.724582000 -0500
@@ -8,6 +8,6 @@
 Description: SignonPlugins
 Version: PROJECT_VERSION
 Requires: libsignon-qt
-Libs.private: -L/usr/lib  -lQtCore
+Libs.private: -L/usr/@LIB@  -lQtCore
 Cflags:  -I${includedir}/signon-plugins -I${includedir}/signon-plugins/SignOn
 Libs: -lsignon-plugins
diff -Nru signon-8.44.orig/lib/SignOn/libsignon-qt.pc.in signon-8.44/lib/SignOn/libsignon-qt.pc.in
--- signon-8.44.orig/lib/SignOn/libsignon-qt.pc.in	2012-11-18 14:07:28.642814000 -0500
+++ signon-8.44/lib/SignOn/libsignon-qt.pc.in	2012-11-18 14:08:41.896849000 -0500
@@ -7,5 +7,5 @@
 Description: Client library for the Single Sign On daemon, Qt bindings.
 Version: PROJECT_VERSION
 Libs: -L${libdir} -lsignon-qt
-Libs.private: -L/usr/lib  -lQtSql -lQtCore -lpthread  
+Libs.private: -L/usr/@LIB@  -lQtSql -lQtCore -lpthread  
 Cflags: -I${includedir}/signon-qt -I${includedir}/signon-qt/SignOn
diff -Nru signon-8.44.orig/src/remotepluginprocess/remotepluginprocess.h signon-8.44/src/remotepluginprocess/remotepluginprocess.h
--- signon-8.44.orig/src/remotepluginprocess/remotepluginprocess.h	2012-11-18 14:07:28.676707000 -0500
+++ signon-8.44/src/remotepluginprocess/remotepluginprocess.h	2012-11-18 14:09:45.844795000 -0500
@@ -47,7 +47,7 @@
 }
 
 #ifndef SIGNOND_PLUGINS_DIR
-      #define SIGNOND_PLUGINS_DIR "/usr/lib/signon"
+      #define SIGNOND_PLUGINS_DIR "/usr/@LIB@/signon"
 #endif
 
 #ifndef SIGNON_PLUGIN_PREFIX
diff -Nru signon-8.44.orig/src/signond/signondaemon.h signon-8.44/src/signond/signondaemon.h
--- signon-8.44.orig/src/signond/signondaemon.h	2012-11-18 14:07:28.687703000 -0500
+++ signon-8.44/src/signond/signondaemon.h	2012-11-18 14:10:10.394754000 -0500
@@ -39,7 +39,7 @@
 #include "credentialsaccessmanager.h"
 
 #ifndef SIGNOND_PLUGINS_DIR
-    #define SIGNOND_PLUGINS_DIR "/usr/lib/signon"
+    #define SIGNOND_PLUGINS_DIR "/usr/@LIB@/signon"
 #endif
 
 #ifndef SIGNOND_PLUGIN_PREFIX
