--- a/lib/plugins/signon-plugins-common/signon-plugins-common.pc.in
+++ b/lib/plugins/signon-plugins-common/signon-plugins-common.pc.in
@@ -6,5 +6,5 @@
 Name: signon-plugins-common
 Description: SignonPluginsCommon
 Version: $$PROJECT_VERSION
-Libs.private: -L/usr/lib  -lQtCore
+Libs.private: -L/usr/@LIB@  -lQtCore
 Cflags:  -I${includedir}/signon-plugins -I${includedir}/signon-plugins/SignOn
--- a/lib/plugins/signon-plugins.pc.in
+++ b/lib/plugins/signon-plugins.pc.in
@@ -8,6 +8,6 @@
 Description: SignonPlugins
 Version: $$PROJECT_VERSION
 Requires: libsignon-qt
-Libs.private: -L/usr/lib  -lQtCore
+Libs.private: -L/usr/@LIB@  -lQtCore
 Cflags:  -I${includedir}/signon-plugins -I${includedir}/signon-plugins/SignOn
 Libs: -lsignon-plugins
--- a/src/plugins/example/exampleplugin.pro
+++ b/src/plugins/example/exampleplugin.pro
@@ -1,7 +1,7 @@
 
 TEMPLATE = lib
 TARGET = exampleplugin
-DESTDIR = lib/signon
+DESTDIR = @LIB@/signon
 QT += core
 
 CONFIG += plugin \
@@ -21,5 +21,5 @@
 QMAKE_CLEAN += libexample.so
 headers.files = $$HEADERS
 
-target.path  = $${INSTALL_PREFIX}/lib/signon
+target.path  = $${INSTALL_PREFIX}/@LIB@/signon
 INSTALLS = target
--- a/src/remotepluginprocess/remotepluginprocess.h
+++ b/src/remotepluginprocess/remotepluginprocess.h
@@ -47,7 +47,7 @@
 }
 
 #ifndef SIGNOND_PLUGINS_DIR
-      #define SIGNOND_PLUGINS_DIR "/usr/lib/signon"
+      #define SIGNOND_PLUGINS_DIR "/usr/@LIB@/signon"
 #endif
 
 #ifndef SIGNON_PLUGIN_PREFIX
--- a/src/signond/signondaemon.h
+++ b/src/signond/signondaemon.h
@@ -39,7 +39,7 @@
 #include "credentialsaccessmanager.h"
 
 #ifndef SIGNOND_PLUGINS_DIR
-    #define SIGNOND_PLUGINS_DIR "/usr/lib/signon"
+    #define SIGNOND_PLUGINS_DIR "/usr/@LIB@/signon"
 #endif
 
 #ifndef SIGNOND_PLUGIN_PREFIX
