diff -Nru libdconf-qt-0.0.0.orig/lib/CMakeLists.txt libdconf-qt-0.0.0/lib/CMakeLists.txt
--- libdconf-qt-0.0.0.orig/lib/CMakeLists.txt	2012-08-18 18:38:11.067411074 -0400
+++ libdconf-qt-0.0.0/lib/CMakeLists.txt	2012-08-18 18:38:26.464409993 -0400
@@ -43,7 +43,7 @@
 #
 set (PREFIXDIR "${CMAKE_INSTALL_PREFIX}")
 set (EXEC_PREFIX "${CMAKE_INSTALL_PREFIX}")
-set (LIBDIR "${CMAKE_INSTALL_PREFIX}/lib")
+set (LIBDIR "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}")
 set (INCLUDEDIR "${CMAKE_INSTALL_PREFIX}/include")
 
 configure_file (${LIB_NAME}.pc.cmake ${CMAKE_CURRENT_BINARY_DIR}/${LIB_NAME}.pc)
