--- a/launcher/CMakeLists.txt
+++ b/launcher/CMakeLists.txt
@@ -91,8 +91,10 @@
 #
 add_executable (launcher StandaloneLauncher.cpp)
 add_dependencies (launcher launcher-lib)
-target_link_libraries (launcher launcher-lib unity-shared)
+pkg_check_modules (CACHED_DBUS_GLIB REQUIRED dbus-glib-1)
+target_link_libraries (launcher launcher-lib unity-shared ${CACHED_DBUS_GLIB_LIBRARIES})
 
 add_executable (switcher StandaloneSwitcher.cpp)
 add_dependencies (switcher switcher-lib launcher-lib)
-target_link_libraries (switcher switcher-lib launcher-lib unity-shared)
+pkg_check_modules (CACHED_DBUS_GLIB REQUIRED dbus-glib-1)
+target_link_libraries (switcher switcher-lib launcher-lib unity-shared ${CACHED_DBUS_GLIB_LIBRARIES})
--- a/panel/CMakeLists.txt
+++ b/panel/CMakeLists.txt
@@ -47,4 +47,5 @@
 #
 add_executable (panel StandalonePanel.cpp)
 add_dependencies (panel panel-lib)
-target_link_libraries (panel panel-lib unity-shared)
+pkg_check_modules (CACHED_DBUS_GLIB REQUIRED dbus-glib-1)
+target_link_libraries (panel panel-lib unity-shared ${CACHED_DBUS_GLIB_LIBRARIES})
