--- a/services/CMakeLists.txt
+++ b/services/CMakeLists.txt
@@ -49,7 +49,7 @@
 
 add_executable(unity-panel-service ${PANEL_SOURCES})
 target_link_libraries(unity-panel-service ${LIBS})
-install(TARGETS unity-panel-service DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/unity/)
+install(TARGETS unity-panel-service DESTINATION ${CMAKE_INSTALL_PREFIX}/libexec/)
 
 configure_file(com.canonical.Unity.Panel.Service.service.cmake ${CMAKE_CURRENT_BINARY_DIR}/com.canonical.Unity.Panel.Service.service)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/com.canonical.Unity.Panel.Service.service DESTINATION ${CMAKE_INSTALL_PREFIX}/share/dbus-1/services)
--- a/services/com.canonical.Unity.Panel.Service.service.cmake
+++ b/services/com.canonical.Unity.Panel.Service.service.cmake
@@ -1,3 +1,3 @@
 [D-BUS Service]
 Name=com.canonical.Unity.Panel.Service
-Exec=${CMAKE_INSTALL_PREFIX}/lib/unity/unity-panel-service
+Exec=${CMAKE_INSTALL_PREFIX}/libexec/unity-panel-service
--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -1,7 +1,7 @@
 #
 # Some unity tools
 #
-install(FILES makebootchart.py PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/unity/)
+install(FILES makebootchart.py PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE DESTINATION ${CMAKE_INSTALL_PREFIX}/libexec/)
 
 configure_file (${CMAKE_CURRENT_SOURCE_DIR}/unity.cmake ${CMAKE_BINARY_DIR}/bin/unity)
 install(FILES ${CMAKE_BINARY_DIR}/bin/unity PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ
--- a/tools/unity.cmake
+++ b/tools/unity.cmake
@@ -27,17 +27,26 @@
 import subprocess
 import sys
 import time
+import platform
+
+if platform.architecture()[0] == "64bit":
+  if platform.linux_distribution()[0] == "Fedora":
+    library_dir = "lib64"
+  else:
+    library_dir = "lib"
+else:
+  library_dir = "lib"
 
 home_dir = os.path.expanduser("~%s" % os.getenv("SUDO_USER"))
 supported_prefix = "/usr/local"
 
 well_known_local_path = ("%s/share/locale/*/LC_MESSAGES/*unity*" % supported_prefix,
                          "%s/share/man/man1/*unity*" % supported_prefix,
-                         "%s/lib/*unity*" % supported_prefix,
+                         "%s/%s/*unity*" % (supported_prefix, library_dir),
                          "%s/share/dbus-1/services/*Unity*" % supported_prefix,
                          "%s/bin/*unity*" % supported_prefix,
                          "%s/include/Unity*"  % supported_prefix,
-                         "%s/lib/pkgconfig/unity*"  % supported_prefix,
+                         "%s/%s/pkgconfig/unity*"  % (supported_prefix, library_dir),
                          "%s/.compiz-1/*/*networkarearegion*" % home_dir,
                          "%s/.config/compiz-1/gsettings/schemas/*networkarearegion*" % home_dir,
                          "%s/.gconf/schemas/*networkarearegion*" % home_dir,
@@ -47,8 +56,8 @@
                          "%s/share/ccsm/icons/*/*/*/*unity*" % supported_prefix,
                          "%s/share/unity" % supported_prefix,
                          "%s/.compiz-1/unity*" % home_dir,
-                         "%s/lib/*nux*"  % supported_prefix,
-                         "%s/lib/pkgconfig/nux*"  % supported_prefix,
+                         "%s/%s/*nux*"  % (supported_prefix, library_dir),
+                         "%s/%s/pkgconfig/nux*"  % (supported_prefix, library_dir),
                          "%s/include/Nux*"  % supported_prefix
                          )
 
--- a/UnityCore/CMakeLists.txt
+++ b/UnityCore/CMakeLists.txt
@@ -132,8 +132,8 @@
 
 install (TARGETS ${CORE_LIB_NAME}
          RUNTIME DESTINATION bin
-         ARCHIVE DESTINATION lib
-         LIBRARY DESTINATION lib
+         ARCHIVE DESTINATION lib${LIB_SUFFIX}
+         LIBRARY DESTINATION lib${LIB_SUFFIX}
         )
 
 #
@@ -145,8 +145,8 @@
 # PkgConfig file
 #
 set (EXEC_PREFIX "${CMAKE_INSTALL_PREFIX}")
-set (LIBDIR "${CMAKE_INSTALL_PREFIX}/lib")
+set (LIBDIR "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}")
 set (INCLUDEDIR "${CMAKE_INSTALL_PREFIX}/include")
 
 configure_file (unity-core.pc.cmake ${CMAKE_CURRENT_BINARY_DIR}/${CORE_LIB_NAME}.pc @ONLY)
-install (FILES ${CMAKE_CURRENT_BINARY_DIR}/${CORE_LIB_NAME}.pc DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/pkgconfig)
+install (FILES ${CMAKE_CURRENT_BINARY_DIR}/${CORE_LIB_NAME}.pc DESTINATION ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/pkgconfig)
--- a/UnityCore/DBusIndicators.cpp
+++ b/UnityCore/DBusIndicators.cpp
@@ -107,7 +107,7 @@
 
     // This is obviously hackish, but this part of the code is mostly hackish...
     // Let's attempt to run it from where we expect it to be
-    std::string cmd = PREFIXDIR + std::string("/lib/unity/unity-panel-service");
+    std::string cmd = PREFIXDIR + std::string("/libexec/unity-panel-service");
     LOG_WARN(logger) << "Couldn't load panel from installed services, "
                      << "so trying to load panel from known location: " << cmd;
 
