diff -Nru indicator-session-0.3.96.orig/src/apt-watcher.c indicator-session-0.3.96/src/apt-watcher.c
--- indicator-session-0.3.96.orig/src/apt-watcher.c	2012-07-12 03:01:21.129923880 -0400
+++ indicator-session-0.3.96/src/apt-watcher.c	2012-07-12 03:02:23.322672788 -0400
@@ -237,9 +237,9 @@
 	  g_free(dialog_line);
   } 
   else{
-    if (!g_spawn_command_line_async("update-manager", &error))
+    if (!g_spawn_command_line_async("gpk-update-viewer", &error))
     {
-      g_warning("Unable to show update-manager: %s", error->message);
+      g_warning("Unable to show gpk-update-viewer: %s", error->message);
       g_error_free(error);
     }
   }   
