--- a/configure.ac
+++ b/configure.ac
@@ -121,7 +121,7 @@
 MAINTAINER_CFLAGS=""
 AS_IF([test "x$enable_maintainer_flags" = "xyes" && test "x$GCC" = "xyes"],
       [
-        MAINTAINER_CFLAGS="-Werror -Wall -Wcast-align -Wno-uninitialized -Wempty-body -Wformat-security -Winit-self"
+        MAINTAINER_CFLAGS="-Wall -Wcast-align -Wno-uninitialized -Wempty-body -Wformat-security -Winit-self"
       ]
 )
 
