diff -Nru compiz-0.9.8.2.orig/cmake/base.cmake compiz-0.9.8.2/cmake/base.cmake
--- compiz-0.9.8.2.orig/cmake/base.cmake	2012-09-14 09:12:03.177518326 -0400
+++ compiz-0.9.8.2/cmake/base.cmake	2012-09-14 09:12:08.054506524 -0400
@@ -80,13 +80,13 @@
 
 # add install prefix to pkgconfig search path if needed
 string (REGEX REPLACE "([\\+\\(\\)\\^\\\$\\.\\-\\*\\?\\|])" "\\\\\\1" PKGCONFIG_REGEX ${CMAKE_INSTALL_PREFIX})
-set (PKGCONFIG_REGEX ".*${PKGCONFIG_REGEX}/lib/pkgconfig:${PKGCONFIG_REGEX}/share/pkgconfig.*")
+set (PKGCONFIG_REGEX ".*${PKGCONFIG_REGEX}/lib${LIB_SUFFIX}/pkgconfig:${PKGCONFIG_REGEX}/share/pkgconfig.*")
 
 if (NOT "$ENV{PKG_CONFIG_PATH}" MATCHES "${PKGCONFIG_REGEX}")
     if ("" STREQUAL "$ENV{PKG_CONFIG_PATH}")
-	set (ENV{PKG_CONFIG_PATH} "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig")
+	set (ENV{PKG_CONFIG_PATH} "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig")
     else ()
 	set (ENV{PKG_CONFIG_PATH}
-	    "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig:$ENV{PKG_CONFIG_PATH}")
+	    "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig:$ENV{PKG_CONFIG_PATH}")
     endif ()
 endif ()
diff -Nru compiz-0.9.8.2.orig/cmake/FindCompiz.cmake compiz-0.9.8.2/cmake/FindCompiz.cmake
--- compiz-0.9.8.2.orig/cmake/FindCompiz.cmake	2012-09-14 09:12:03.177518326 -0400
+++ compiz-0.9.8.2/cmake/FindCompiz.cmake	2012-09-14 09:12:08.056506519 -0400
@@ -55,16 +55,16 @@
 
 	# add install prefix to pkgconfig search path if needed
         string (REGEX REPLACE "([\\+\\(\\)\\^\\\$\\.\\-\\*\\?\\|])" "\\\\\\1" PKGCONFIG_REGEX ${CMAKE_INSTALL_PREFIX})
-	set (PKGCONFIG_REGEX ".*${PKGCONFIG_REGEX}/lib/pkgconfig:${PKGCONFIG_REGEX}/share/pkgconfig.*")
+	set (PKGCONFIG_REGEX ".*${PKGCONFIG_REGEX}/lib${LIB_SUFFIX}/pkgconfig:${PKGCONFIG_REGEX}/share/pkgconfig.*")
 
 	if (NOT CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 
 	    if (NOT "$ENV{PKG_CONFIG_PATH}" MATCHES "${PKGCONFIG_REGEX}")
 		if ("" STREQUAL "$ENV{PKG_CONFIG_PATH}")
-		    set (ENV{PKG_CONFIG_PATH} "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig")
+		    set (ENV{PKG_CONFIG_PATH} "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig")
 	    	else ()
 		    set (ENV{PKG_CONFIG_PATH}
-		         "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig:$ENV{PKG_CONFIG_PATH}")
+		         "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig:$ENV{PKG_CONFIG_PATH}")
 		endif ()
 	    endif ()
 
diff -Nru compiz-0.9.8.2.orig/compizconfig/libcompizconfig/cmake/FindCompizConfig.cmake compiz-0.9.8.2/compizconfig/libcompizconfig/cmake/FindCompizConfig.cmake
--- compiz-0.9.8.2.orig/compizconfig/libcompizconfig/cmake/FindCompizConfig.cmake	2012-09-14 09:12:02.542519880 -0400
+++ compiz-0.9.8.2/compizconfig/libcompizconfig/cmake/FindCompizConfig.cmake	2012-09-14 09:12:08.057506516 -0400
@@ -53,14 +53,14 @@
 
 	# add install prefix to pkgconfig search path if needed
         string (REGEX REPLACE "([\\+\\(\\)\\^\\\$\\.\\-\\*\\?\\|])" "\\\\\\1" PKGCONFIG_REGEX ${CMAKE_INSTALL_PREFIX})
-	set (PKGCONFIG_REGEX ".*${PKGCONFIG_REGEX}/lib/pkgconfig:${PKGCONFIG_REGEX}/share/pkgconfig.*")
+	set (PKGCONFIG_REGEX ".*${PKGCONFIG_REGEX}/lib${LIB_SUFFIX}/pkgconfig:${PKGCONFIG_REGEX}/share/pkgconfig.*")
 
 	if (NOT "$ENV{PKG_CONFIG_PATH}" MATCHES "${PKGCONFIG_REGEX}")
 	    if ("" STREQUAL "$ENV{PKG_CONFIG_PATH}")
-		set (ENV{PKG_CONFIG_PATH} "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig")
+		set (ENV{PKG_CONFIG_PATH} "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig")
 	    else ()
 		set (ENV{PKG_CONFIG_PATH}
-		    "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig:$ENV{PKG_CONFIG_PATH}")
+		    "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/pkgconfig:${CMAKE_INSTALL_PREFIX}/share/pkgconfig:$ENV{PKG_CONFIG_PATH}")
 	    endif ()
 	endif ()
 
diff -Nru compiz-0.9.8.2.orig/compizconfig/libcompizconfig/cmake/LibCompizConfigCommon.cmake compiz-0.9.8.2/compizconfig/libcompizconfig/cmake/LibCompizConfigCommon.cmake
--- compiz-0.9.8.2.orig/compizconfig/libcompizconfig/cmake/LibCompizConfigCommon.cmake	2012-09-14 09:12:02.541519882 -0400
+++ compiz-0.9.8.2/compizconfig/libcompizconfig/cmake/LibCompizConfigCommon.cmake	2012-09-14 09:12:08.057506516 -0400
@@ -106,7 +106,7 @@
 
 	set_target_properties (
 	    ${bname} PROPERTIES
-	    INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib"
+	    INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}"
 	    COMPILE_FLAGS "${${_BACKEND}_CFLAGSADD}"
 	    LINK_FLAGS "${${_BACKEND}_LDFLAGSADD}"
 	)
@@ -122,7 +122,7 @@
 
     install (
 	TARGETS ${bname}
-	DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/compizconfig/backends
+	DESTINATION ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/compizconfig/backends
     )
 
     if (NOT _COMPIZCONFIG_INTERNAL)
