--- a/postinst/migration-scripts/02_migrate_to_gsettings.py
+++ b/postinst/migration-scripts/02_migrate_to_gsettings.py
@@ -25,7 +25,7 @@
 import os.path
 
 # this should point to the directory where all the .convert files are stored
-CONVERT_PATH = "/usr/lib/compiz/migration/"
+CONVERT_PATH = "/usr/share/compiz/migration/"
 
 def migrate_file(convert_file):
     subprocess.Popen(["gsettings-data-convert", "--file={}{}".format(CONVERT_PATH, convert_file)]).communicate()
